pywebio/platform/tornado.py

```diff
@@ -462,7 +462,7 @@ def server_thread():
             port = int(os.environ.get("PYWEBIO_SCRIPT_MODE_PORT"))
 
         server, port = _setup_server(webio_handler=SingleSessionWSHandler, port=port, host='localhost',
-                                     websocket_max_message_size=parse_file_size('4G'))
+                                     websocket_max_message_size=parse_file_size('200M'))
```

Comment:
Can you try to set  `websocket_max_message_size` to `2G` or `1G` and see if it works ok?

Since developers can't modify this config, `200M` will seem too small in some cases.

Reply:
Yes, I tried to set to 1G, and it was not working

Reply:
OK, thanks